File: 0001 Code Chef/0025 Burger/implementation/0010 nodejs/burger.js
const readline = require('readline');

const rl = readline.createInterface({
    input:process.stdin,
    output:process.stdout
});


rl.question('',(input)=>{
    let test = parseInt(input);
    let count = 0;

    rl.on('line',(input)=>{
        let string_array = input.split(" ");
        let a = parseInt(string_array[0]);
        let b = parseInt(string_array[1]);
        if(a > b){
            console.log(b);
        }else{
            console.log(a);
        }
        count = count + 1;
        if(count == test){
            rl.close()
        }
    })
})

File: 0001 Code Chef/0025 Burger/implementation/0008 python3/burger.py
if __name__ == "__main__":
    test = int(input())
    while(test > 0):
        string_input = input()
        string_array = string_input.split(" ")
        a = int(string_array[0])
        b = int(string_array[1])
        if(a > b):
            print(b)
        else:
            print(a)
        test = test - 1;
